src/sagemaker/session.py

```diff
@@ -638,11 +639,15 @@ def create_model_from_job(self, training_job_name, name=None, role=None, primary
                 Default: use VpcConfig from training job.
                 * 'Subnets' (list[str]): List of subnet ids.
                 * 'SecurityGroupIds' (list[str]): List of security group ids.
+            tags(List[dict[str, str]]): Optional. The list of tags to add to the model. For more, see
+                https://docs.aws.amazon.com/sagemaker/latest/dg/API_Tag.html.
```

Comment:
add that this defaults to the job's tags. (might also need to be updated in other docstrings.) alternatively, you could default to `self.tags` before this method is called in the estimator.

Reply:
the second option 👌🏽